src/app/batch_txn_tool/batch_txn_tool.ml

```diff
@@ -387,33 +390,23 @@ let output_there_and_back_cmds =
             transactions, if this is not present then we use the env var \
             MINA_PRIVKEY_PASS"
          (optional string)
-     and config_file = Cli_lib.Flag.config_files
      and graphql_target_node_option =
        flag "--graphql-target-node" ~aliases:[ "graphql-target-node" ]
          ~doc:
            "URL The graphql node to send graphl commands to.  must be in \
             format `<ip>:<port>`.  default is `127.0.0.1:3085`"
          (optional string)
-     and minimum_user_command_fee_opt = Cli_lib.Flag.fee_common in
```

Comment:
this code removes the runtime overwrite and sticks to the compile time configuration.